site/monitoring.xml

```diff
@@ -0,0 +1,105 @@
+<!ENTITY % entities SYSTEM "rabbit.ent">
+%entities;
+]>
+<!--
+Copyright (c) 2007-2017 Pivotal Software, Inc.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License, 
+Version 2.0 (the "License”); you may not use this file except in compliance 
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:doc="http://www.rabbitmq.com/namespaces/ad-hoc/doc">
+  <head>
+    <title>Monitoring</title>
+  </head>
+  <body show-in-this-page="true">
+    <p class="intro">This document is intended to guide you on which RabbitMQ and system metrics
+      are most important to monitor. Monitoring your RabbitMQ installation is an effective means
+      to intercept issues before they affect the rest of your environment and, eventually, your
+      users.
+    </p>
+    <doc:section name="system-metrics">
+      <doc:heading>System Metrics</doc:heading>
+      <ul>
+        <li>CPU (idle, user, system, iowait)</li>
+        <li>System load average (<code>/proc/loadavg</code>)</li>
+        <li>Memory (free, cached, buffered)</li>
+        <li>Disk I/O (reads &amp; writes per unit time, I/O wait percentages)</li>
+        <li>Free Disk Space</li>
+        <li>File descriptors used by <code>beam.smp</code> vs available</li>
+        <li>Network throughput (bytes received, bytes sent)</li>
+        <li><a href="https://www.kernel.org/doc/Documentation/sysctl/vm.txt">VM</a> statistics (dirty page flushes, writeback volume)</li>
+      </ul>
+    </doc:section>
+    <doc:section name="rabbitmq-metrics">
+      <doc:heading>RabbitMQ Metrics</doc:heading>
+      <p>
+        The RabbitMQ <a href="management.html">management plugin</a> provides a starting point for
+        monitoring RabbitMQ metrics. One limitation, however, is that only up to one day's worth of
+        metrics are stored. Storing historical metrics can be an important tool to determine the root
+        cause of issues affecting your users or to plan for future capacity.
+      </p>
+      <p>
+        RabbitMQ metrics are made available through the <a href="management.html#http-api">HTTP API</a>
+        or via several <a href="man/rabbitmqctl.1.man.html"><code>rabbitmqctl</code></a> commands. It is
+        recommended to collect metrics at 60 second intervals because more frequent collection may place
+        too much load on the RabbitMQ server and negatively affect performance.
+      </p>
+      <p>
+        <table>
+          <thead>
+            <tr><td>Metric</td><td>JSON field name</td></tr>
```

Comment:
Maybe it's worth mentioning which HTTP endpoint(s) those metrics belong to.

Reply:
Got it, added the following:

```
via the <a href="https://cdn.rawgit.com/rabbitmq/rabbitmq-management/&version-server-tag;/priv/www/api/index.html"><code>api/queues/<i>vhost</i>/<i>qname</i></code></a>
```